### PR TITLE
add python rencode

### PR DESCRIPTION
**Pull request by thesmo**
Please let me know if this conforms. I used the template for a python module.

I would like to commit more of these. Do I do one big pull request or 1 at a time for review?

### CODE REVIEW

File: mingw-w64-python-rencode/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Maintainer: Smo <smo@xpra.org>
+
+_realname=rencode
+pkgbase=mingw-w64-python-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-${_realname}"  
+"${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+pkgver=1.0.5
+pkgrel=1
+pkgdesc="Python module for fast (basic) object serialization similar to bencode"
```
Comment (Alexpux) [inline mingw-w64-python-rencode/PKGBUILD]: Add ````(mingw-w64)```` to the end of description

File: mingw-w64-python-rencode/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-python2"
+             "${MINGW_PACKAGE_PREFIX}-python3"
+             "${MINGW_PACKAGE_PREFIX}-python3-setuptools"
+             "${MINGW_PACKAGE_PREFIX}-python2-setuptools"
+             "${MINGW_PACKAGE_PREFIX}-cython"
+             "${MINGW_PACKAGE_PREFIX}-cython2")
+
+options=('staticlibs' 'strip' '!debug')
+
+source=("https://github.com/aresch/rencode/archive/v${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-python-rencode/PKGBUILD]: Download into named archive

Comment (Alexpux) [reply mingw-w64-python-rencode/PKGBUILD]: Look other PKGBUILDs and you will see how to do it

Comment (Adsun701) [reply mingw-w64-python-rencode/PKGBUILD]: No. "${_realname}-${pkgver}.tar.gz::https://github.com/aresch/rencode/archive/v${pkgver}.tar.gz" will work.

**Pull request merged**